src/atoms/visual-story/web-story-page-components/web-story-page-components.tsx

```diff
@@ -24,7 +30,8 @@ const WebStoryPageComponentsBase = ({ card, config, story }: WebStoryPageCompone
 
   return (
     <Fragment>
-      {imageElement && (
+      {videoUrl && <VideoWebStory videoUrl={videoUrl} config={config} imageElement={imageElement} />}
```

Comment:
no need to pass config this way. `VideoWebStory` component can get config from context